tests/e2e/commandse2e/blockchain/deploy/suite.go

```diff
@@ -41,6 +43,127 @@ var _ = ginkgo.Describe("[Blockchain Deploy Flags]", ginkgo.Ordered, func() {
 		gomega.Expect(err).Should(gomega.BeNil())
 	})
 
+	ginkgo.It("HAPPY PATH: local deploy with avalanchego path set", func() {
+		avalanchegoPath := "tests/e2e/assets/mac/avalanchego"
+		if runtime.GOOS == "linux" {
+			avalanchegoPath = "tests/e2e/assets/linux/avalanchego"
+		}
+		testFlags := utils.TestFlags{
+			"avalanchego-path": avalanchegoPath,
+		}
+		output, err := utils.TestCommand(utils.BlockchainCmd, "deploy", blockchainCmdArgs, globalFlags, testFlags)
+		gomega.Expect(output).Should(gomega.ContainSubstring(fmt.Sprintf("AvalancheGo path: %s", avalanchegoPath)))
+		gomega.Expect(output).Should(gomega.ContainSubstring("L1 is successfully deployed on Local Network"))
+		gomega.Expect(err).Should(gomega.BeNil())
+	})
+
+	ginkgo.It("HAPPY PATH: local deploy convert only", func() {
+		testFlags := utils.TestFlags{
+			"convert-only": true,
+		}
+		output, err := utils.TestCommand(utils.BlockchainCmd, "deploy", blockchainCmdArgs, globalFlags, testFlags)
+		gomega.Expect(output).Should(gomega.ContainSubstring("To finish conversion to sovereign L1, create the corresponding Avalanche node(s) with the provided Node ID and BLS Info"))
+		gomega.Expect(output).Should(gomega.ContainSubstring("Ensure that the P2P port is exposed and 'public-ip' config value is set"))
+		gomega.Expect(output).Should(gomega.ContainSubstring("Once the Avalanche Node(s) are created and are tracking the blockchain, call `avalanche contract initValidatorManager testSubnet` to finish conversion to sovereign L1"))
+		gomega.Expect(output).ShouldNot(gomega.ContainSubstring("L1 is successfully deployed on Local Network"))
+		gomega.Expect(err).Should(gomega.BeNil())
+	})
+
+	ginkgo.It("HAPPY PATH: generate node id ends in convert only", func() {
+		testFlags := utils.TestFlags{
+			"generate-node-id": true,
+		}
+		output, err := utils.TestCommand(utils.BlockchainCmd, "deploy", blockchainCmdArgs, globalFlags, testFlags)
+		gomega.Expect(output).Should(gomega.ContainSubstring("To finish conversion to sovereign L1, create the corresponding Avalanche node(s) with the provided Node ID and BLS Info"))
+		gomega.Expect(output).Should(gomega.ContainSubstring("Ensure that the P2P port is exposed and 'public-ip' config value is set"))
+		gomega.Expect(output).Should(gomega.ContainSubstring("Once the Avalanche Node(s) are created and are tracking the blockchain, call `avalanche contract initValidatorManager testSubnet` to finish conversion to sovereign L1"))
+		gomega.Expect(output).ShouldNot(gomega.ContainSubstring("L1 is successfully deployed on Local Network"))
+		gomega.Expect(err).Should(gomega.BeNil())
+	})
+
```

Comment:
not testing bootstrap endpoint because we are already testing it in etna test suite